std/node/module.ts

```diff
@@ -19,6 +19,12 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
```

Comment:
I think calling this `require.ts` is better. In this context, the name "require", for me, conjures immediately the notion of Node's require; whereas "module" is much more ambiguous but has a more ES-Modules vibe to it (which is not what we want to connote)

Reply:
Hmm but on the other hand do we want to expose other stuff from this module that are available from Node's `module` built-in module?

Since `std/node` is intended for Node.js compatibility, IMO it might be better to comply...

Reply:
ah, right. make sense.